src/main/java/org/jvnet/hudson/plugins/port_allocator/PortAllocator.java

```diff
@@ -36,14 +40,21 @@ private PortAllocator(PortType[] ports){
         this.ports = ports;
     }
 
+    @DataBoundConstructor
+    public PortAllocator(String pool) {
```

Comment:
No, I mean in the config.xml of the job.
To check if it's backward compatible: 1) install the previous version of the plugin and configure this build wrapper in a job, then 2) upgrade the plugin to this snapshot and check if the job configuration is still correct and nothing is missing